django/db/backends/postgresql_psycopg2/operations.py

```diff
@@ -212,3 +212,17 @@ def return_insert_id(self):
     def bulk_insert_sql(self, fields, num_values):
         items_sql = "(%s)" % ", ".join(["%s"] * len(fields))
         return "VALUES " + ", ".join([items_sql] * num_values)
+
+    def value_to_db_date(self, value):
+        return value
+
+    def value_to_db_datetime(self, value):
+        return value
+
+    def value_to_db_time(self, value):
+        return value
+
+    def value_to_db_ipaddress(self, value):
+        if value:
+            return Inet(value)
+        return None
```

Comment:
`return None` not strictly needed, but if you think it adds clarity I'm okay with it.